addons/survey/static/src/js/survey_form.js

```diff
@@ -430,9 +433,31 @@ publicWidget.registry.SurveyFormWidget = publicWidget.Widget.extend({
         self.$(selectorsToFadeout.join(',')).fadeOut(this.fadeInOutDelay, function () {
             resolveFadeOut();
         });
+        // Background management - Will the background change between current and next/previous questions ?
+        var refreshBackground = self.options.reloadBackgroundOnNext || options.goBack;
+        if (refreshBackground) {
+            // If we make a transition, hide previous background by setting 'background overlay' opacity = 1
+            $('div#wrapwrap').css("box-shadow", "inset 0 0 0 10000px rgba(255,255,255,1)");
```

Comment:
Isn't it possible to make a class out of this?
Would simplify this part and give a name to it since it's used a couple times.
Something like "o_survey_background_transition" ?